.github/workflows/libs/gha.libsonnet

```diff
@@ -55,6 +55,12 @@
     run: 'git config --system core.longpaths true'
   },
 
+  git_set_core_autocrlf(mode): {
```

Comment:
comment please, why we need that (can say it's for windows, with a vague explanation
of why we need it).